docs/source/item-validation.rst

```diff
@@ -193,6 +221,37 @@ as a `dict`:
         OtherItem: '/path/to/otheritem_schema.json',
     }
 
+.. _SPIDERMON_VALIDATION_CERBERUS:
+
+SPIDERMON_VALIDATION_CERBERUS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``None``
+
+A `list` containing the local path of the item schema.
+
+.. code-block:: python
+
+    # settings.py
+
+    SPIDERMON_VALIDATION_CERBERUS: [
+        '/path/to/schema.json',
+        'http://example.com/mycerberusschema',
+        {"Field": {"type": "number", "required":True}}
```

Comment:
Is this allowed? According to the docs, this list contains the path for the schemas, not the schema itself.

Reply:
Yes, it is allowed. I have tried that. I can share you the `quotes` spider that is working perfectly if you wish to check it out. 
Raw Schema works because these are basically dicts that are being passed in the pipelines which are not same as string_types hence they get directly passed down to the CerberusValidator() class. I felt its best implemented this way as I described in the call as well.

Testing instructions:
Clone https://github.com/vipulgupta2048/testing_quotes
install the cerberus branch of vipulgupta2048/spidermon in virtualvenv
Uncomment whichever method you like to test in settings.py and run scrapy crawl <spider_name>